### PR TITLE
declutter spyglass logging

### LINKED ISSUE #9982: spyglass: log with more discretion
Currently, I see:

```
# ./bin/deck --help
INFO[0000] Spyglass registered viewer build-log-viewer with title Build Log. 
INFO[0000] Spyglass registered viewer junit-viewer with title JUnit. 
INFO[0000] Spyglass registered viewer metadata-viewer with title Metadata. 
Usage of ./bin/deck:
  -alsologtostderr
    	log to standard error as well as files
...
```

That doesn't seem right :)

/cc @amwat @BenTheElder

### PR DESCRIPTION
**Pull request by ibzib**
fixes #9982

### CODE REVIEW

File: prow/spyglass/viewers/viewers.go [hunk 1]
```diff
@@ -146,7 +146,6 @@ func RegisterViewer(viewerName string, metadata ViewMetadata, handler ViewHandle
 	}
 	viewHandlerReg.reg[viewerName] = handler
 	viewMetadataReg.reg[viewerName] = metadata
-	logrus.Infof("Spyglass registered viewer %s with title %s.", viewerName, metadata.Title)
```
Comment (Katharine) [inline prow/spyglass/viewers/viewers.go]: I think these logs are pretty useful: they're one-offs on launch, and I believe are also the only way to know that your viewer was registered if it fails during startup.

Comment (BenTheElder) [inline prow/spyglass/viewers/viewers.go]: maybe change the log level instead of removing it entirely? this seems useful to know

**Pull request closed without merging**